schema/pharmgkbSchema.mcf

```diff
@@ -0,0 +1,333 @@
+Node: dcid:ChemicalCompoundGeneAssociation
+name: "ChemicalCompoundGeneAssociation"
+typeOf: schema:Class
+subClassOf: schema:Intangible
+description: "The association of a chemical compound with a gene."
+
+Node: dcid:compoundID
+name: "compoundID"
+typeOf: schema:Property
+rangeIncludes: dcs:Drug
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:geneID
+name: "geneID"
+typeOf: schema:Property
+rangeIncludes: dcs:Gene
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:pubMedID
+name: "pubMedID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:isPharmacokineticRelationship
+name: "isPharmacokineticRelationship"
+typeOf: schema:Property
+rangeIncludes: schema:Boolean
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+description: "True if the pair of entities was found in a pharmacokinetic pathway on PharmGKB, or if the Variant Annotation or VIP was annotated with PK in some manner."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:isPharmacodynamicRelationship
+name: "isPharmacodynamicRelationship"
+typeOf: schema:Property
+rangeIncludes: schema:Boolean
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+description: "True if the pair of entities was found in a pharmacodynamic pathway on PharmGKB, or if the Variant Annotation or VIP was annotated with PD in some manner."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:relationshipAssociationType
+name: "relationshipAssociationType"
+typeOf: schema:Property
```

Comment:
Please add descriptions to all properties whenever possible

Reply:
+1